logstash-core/lib/logstash/patches/clamp.rb

```diff
@@ -0,0 +1,69 @@
+require 'clamp'
+require 'logstash/environment'
+
+module Clamp
+  module Attribute
+    class Instance
+      def default_from_environment
```

Comment:
Do we need this? IIRC clamp only does stuff with the environment if you set `:environment_variable => "NAME"` in the option setting, like `option "--foo", ..., :environment_variable => "FOO"`. The default behavior is that no env vars are used on an option.?